--- md5sums     2024-02-29 15:35:04.000000000 +0000
+++ md5sums    2024-09-01 15:54:34.436962032 +0000
@@ -14,7 +14,7 @@
 2745c50f6f4e395e7b7d52f902d075bf  diffutils-3.10.tar.xz
 6b4f18a33873623041857b4963641ee9  e2fsprogs-1.47.0.tar.gz
 79ad698e61a052bea79e77df6a08bc4b  elfutils-0.190.tar.bz2
-bd169cb11f4b9bdfddadf9e88a5c4d4b  expat-2.6.0.tar.xz
+0cb75c8feb842c0794ba89666b762a2d  expat-2.6.2.tar.xz
 00fce8de158422f5ccd2666512329bd2  expect5.45.4.tar.gz
 26b2a96d4e3a8938827a1e572afd527a  file-5.45.tar.gz
 4a4a547e888a944b2f3af31d789a1137  findutils-4.9.0.tar.xz

