--- wget-list-systemd   2024-02-29 15:35:04.000000000 +0000
+++ wget-list-systemd   2024-09-01 15:40:21.048896773 +0000
@@ -14,7 +14,7 @@
 https://ftp.gnu.org/gnu/diffutils/diffutils-3.10.tar.xz
 https://downloads.sourceforge.net/project/e2fsprogs/e2fsprogs/v1.47.0/e2fsprogs-1.47.0.tar.gz
 https://sourceware.org/ftp/elfutils/0.190/elfutils-0.190.tar.bz2
-https://prdownloads.sourceforge.net/expat/expat-2.6.0.tar.xz
+https://prdownloads.sourceforge.net/expat/expat-2.6.2.tar.xz
 https://prdownloads.sourceforge.net/expect/expect5.45.4.tar.gz
 https://astron.com/pub/file/file-5.45.tar.gz
 https://ftp.gnu.org/gnu/findutils/findutils-4.9.0.tar.xz
